library/psa_crypto.c

```diff
@@ -1041,18 +1041,30 @@ psa_status_t psa_destroy_key( psa_key_handle_t handle )
         if( status != PSA_SUCCESS )
         {
             (void) psa_crypto_stop_transaction( );
-            /* TOnogrepDO: destroy what can be destroyed anyway */
-            return( status );
+            /* We should still try to destroy the key in the secure
```

Comment:
Did you omit the TODO label on purpose here?

Reply:
That's more of a lack of robustness rather than an outright bug or missing functionality. It is a blurry line though.